api/src/main/java/me/lokka30/treasury/api/economy/currency/CurrencyManager.java

```diff
@@ -65,6 +65,102 @@ public void registerCurrency(@NotNull Currency currency) {
         }
     }
 
+    /**
+     * @param currencies currencies to register
+     * @see #registerCurrency(Currency)
+     */
+    public void registerCurrency(@NotNull Currency @NotNull ... currencies) {
+        Objects.requireNonNull(currencies, "currencies");
+        for (Currency currency : currencies) {
+            registerCurrency(currency);
+        }
+    }
+
+    /**
+     * @param currencies currencies to register
+     * @see #registerCurrency(Currency)
+     */
+    public void registerCurrency(@NotNull Iterable<@NotNull Currency> currencies) {
```

Comment:
Should this be `registerCurrencies`?

Reply:
why should it be

Reply:
Because it allows to register several currencies at ones